### DISCUSSION TITLE
Ajout deux propriétés à EntrantRechercherFormulairesSIS

### DISCUSSION THREAD

**Discussion by givanretqc** [MTESSDev/FRW#133] [Annonces]
Bonjour,

Je fais partie de l'équipe du P541, un projet effectué à Retraite Québec qui utilise cette ressource logicielle. On m'a chargé de modifications nécessitant entre autres ajouter deux critères de recherche à l'obtention de formulaires via la fonction ObtenirFormulairesIndividu, donc on m'a demandé de faire ajouter deux propriétés à l'objet entrant EntrantRechercherFormulairesSIS:

-CodeTypeUtilisateurFormulaireWeb (string)
-ValeurIdentifiantUtilisateurFormulaireWeb (string)

Si ce n'est pas le bon moyen pour effectuer ce genre de demande informez moi de la procédure correcte.

Merci d'avance et bonne journée!

**Comment by vinnyboii** [comment]
Bonjour,

Actuellement, nous avons le paramètre `X-IdentifiantUtilisateur` qui permet de spécifier l'utilisateur pour lequel on veut obtenir les formulaires.

Pouvez-vous nous préciser votre besoin par rapport au `CodeTypeUtilisateurFormulaireWeb` ? Quels sont vos cas d'utilisation ?

Merci et bonne journée

Vincent
